app/controllers/endpoints_controller.rb

```diff
@@ -37,11 +39,26 @@ def permitted_params
     )
   end
 
+  def update_params
```

Comment:
for updating, the passing params are `certificate_id` and `ssl_policy`

https://github.com/degica/barcelona-cli/blob/4509ab33cd2252ff0660a2d5f187767e7d8107b7/cmd/endpoint.go#L159-L163